shared/desktop/app/main-window.desktop.tsx

```diff
@@ -91,7 +91,8 @@ const setupWindowEvents = (win: Electron.BrowserWindow) => {
 }
 
 const changeDock = (show: boolean) => {
-  const dock = Electron.app.dock
+  const _dock = Electron.app.dock
+  const dock = _dock as typeof _dock | undefined
```

Comment:
the electron typing is incorrect and its a little annoying to have to keep patching it so lets just cast it